tests/policies/test_interactive.py

```diff
@@ -10,44 +11,107 @@
 from imitation.policies import interactive
 
 ENVS = [
-    "Pong-v4",
+    "CartPole-v0",
```

Comment:
Consider using seals version instead which has a fixed termination condition (500 timesteps). CartPole-v0 in Gym may terminate quickly (as soon as pole drops below threshold) which might cause test to be more vacuous than intended.
```suggestion
    "seals/CartPole-v0",
```